src/cmd/cli/command/commands.go

```diff
@@ -1229,7 +1229,7 @@ func getProvider(ctx context.Context, loader *compose.Loader) (cliClient.Provide
 
 	projName, err := cliClient.LoadProjectNameWithFallback(ctx, loader, provider)
 	if err != nil {
-		return nil, err
+		term.Debug("unable to load project name:", err)
```

Comment:
I don't think it is a hack, this was the original intended behaviour:
https://github.com/DefangLabs/defang/commit/c9be97991ef5b9f175be2f0e3d2f12de056a143d#diff-54c090d7a55cbcb94d557b01cf00846e8cbe18d69705fc322ecc61de4b78b833R1138